src/Chapter2.hs

```diff
@@ -740,8 +788,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
+contains :: Int -> [[Int]] -> [[Int]]
+contains n list = filter (\l -> elem n l) list
```

Comment:
A few opportunities of eta-reduction are here :slightly_smiling_face: 
You can notice that you can eta-reduce lambda with `elem` because they end on the same argument. And you can also eta-reduce the `list` variable.